docs/listeners/related-models.rst

```diff
@@ -1,26 +1,21 @@
 Related Models
 ==============
 
-If you are used to bake or CakePHP scaffolding you might want to have some
-control over the data it is sent to the view for filling select boxes for
-associated models.
+If you are used to bake or CakePHP scaffolding you might want to have some control over the data it sends to the view
+for filling select boxes for associated models.
 
 Introduction
 ------------
 
-CRUD can be configured to return the list of record for all
-related models or just those you want to in a per-action basis.
+Crud can be configured to return the list of records for all related models or just those you want to on a per-action
+basis. By default all related model lists for the main Crud table instance will be fetched, but only for add, edit and
```

Comment:
I'm not sure I fully understand this but I believe this contains one crucial piece of information that maybe should be placed below the `RelatedModels` section as a note. Probably self-evident to you guys but this would help me as a starting user.

> Please note that Related Models does not work on `index` actions.